package.json

```diff
@@ -4,6 +4,6 @@
   "description": "Tools for CI with Moodle",
   "private": true,
   "dependencies": {
-    "vnu-jar": "^17.3.0"
+    "vnu-jar": "18.11.5"
```

Comment:
I think it's ok if we start providing a lock file, as you comment in the issue. Cannot imagine any downside, but better control and safer results. My +1 here.

Reply:
I added commit with lock file.

Reply:
super!